package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "dev": "ts-node-dev --respawn ./src/index.ts",
         "build": "tsc",
-        "start": "node ./dist/index.js",
+        "start": "node .",
```

Comment:
🤔?

Reply:
Oops, My bad, `node .` is equal to `node ./dist/index.js` in this context, because `.` got translated to `${main}`

Reply:
I could revert it if you want

Reply:
my bad i thought you made this change by incident because it is not related to this pr